src/schematics/deploy/builder.ts

```diff
@@ -33,12 +36,15 @@ export default createBuilder<any>(
       context.target.project
     );
 
+    const buildTarget = options.buildTarget || `build:${context.target.project}:production`;
```

Comment:
### Note
I'm not sure this is the best approach. Ideally `ng add @angular/fire` would automatically define `project.architect.deploy.options.buildTarget` in `angular.json`.

This could easily be done here [`schematics / ng-add.ts #L231`](https://github.com/angular/angularfire/blob/master/src/schematics/ng-add.ts#L231).

### Pros
We could make `buildTarget` required and make configuration more obvious for developers.

### Cons
This could cause backwards compatibility issues.

Reply:
Agreed, it should be automatically defined on `ng add`. However, backwards compatibility should be maintained ie a default fallback value.